### ISSUE TITLE
Firefox: Misaligned check boxes 

### ISSUE THREAD

**Issue by Prateek032** [att/rcloud#1957]
This issues is observed on the latest Firefox version (46.0).
For example: In setting div

![image](https://cloud.githubusercontent.com/assets/10941328/15036653/e23cdf16-12af-11e6-8d39-e673f397cfe4.png)

**Comment by shaneporter** [comment]
It's ok on Firefox 45.0.1, Ubuntu:

![image](https://cloud.githubusercontent.com/assets/2493614/15039432/0861791e-12a5-11e6-8da5-51c237cd55c9.png)

I'll upgrade and take a look.

**Comment by shaneporter** [comment]
I have looked at Firefox 46 checkboxes on Windows, and they are 'normal'. Unable to check on Mac, but regardless of that platform's rendering, we have an issue.

I can't think of an alternative to styling checkboxes across platforms/browsers.

It seems all styling/JavaScript solutions require a particular structure of label and input (there are 2 alternatives as I recall.) I'd have to go through and verify that all are using a particular structure, because if not, they will not be visible.

**Comment by gordonwoodhull** [comment]
Weird, why would they make them so large on one platform only? Of course it is okay in Firefox 46 on OSX.

We always do `<label><input ...></input> Label text</label>`, or I think we always do. That allows you to click on the text to invoke the checkbox. 

There's probably a trivial CSS trick we can do - but if not, I think we can safely chalk this up as a browser bug.

**Comment by shaneporter** [comment]
Yes, as far as I can see, it's a browser issue, since it's not reproducible on other OSs or previous versions of Firefox on Ubuntu.

**Comment by gordonwoodhull** [comment]
This happens also on Chrome and Safari on iOS. 
![image](https://cloud.githubusercontent.com/assets/1366709/15365372/fecf6052-1ced-11e6-9baf-ebfec8be43a8.jpeg)

**Comment by shaneporter** [comment]
Oh dear.
